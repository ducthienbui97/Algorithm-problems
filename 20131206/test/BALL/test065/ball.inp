8 926439 802170 382680 276289
766594 914935 985431 680736 831743 638951 444488 766904 329184 906803 35534 521759 904250 680307 617432 599791 342258 599561 971980 645729 415806 79598 776232 569034 66612 621003 545857 707588 901294 873819 297183 646574 656916 649829 291318 262431 54914 967281 587792 550505 643331 212615 844403 349254 259246 234072 693924 297154 297980 729493 353140 786057 801408 527872 542251 121827 256117 246596 635120 353670 292020 460836 708821 299923 
460182 137099 335759 110126 274697 617187 925474 896074 36978 207251 906392 876085 411633 644279 830199 398709 592361 305221 687385 6389 852036 908938 785958 500592 954654 536581 944371 335240 971791 680374 155171 75748 367226 894032 23154 325881 175073 319775 965451 367943 574543 690592 592205 818088 595744 634451 217414 15408 90288 578175 120509 773078 861510 757867 915823 135299 58754 77873 538096 710609 802119 426431 409073 427030 
67841 558475 592738 204520 445845 413956 495575 456290 577923 230335 438523 254027 734043 7667 217944 164577 924115 652515 944309 382815 327335 343307 899267 950289 523169 859038 30439 359670 829104 795900 273575 399339 907685 194502 736924 815571 776207 393075 192170 127993 675008 907291 263392 616988 158249 815603 168432 533934 161826 273752 11408 768661 784682 326157 420900 88893 486914 216548 154792 151560 108778 108444 981908 801861 
20953 946956 67177 348320 283507 882610 189497 103411 162849 425648 930371 75005 11542 30490 672219 665316 87625 118519 4191 903383 424277 373378 904538 537406 566132 123788 550329 491250 872043 715150 573533 777781 788765 28675 595771 304376 500121 637280 720897 768848 937284 184776 968517 118506 167161 362943 213694 518051 934514 810263 273110 542271 956170 941331 235883 142257 469852 195423 888709 807545 715859 363243 714868 207438 198903 247048 746097 404308 48082 226529 192495 524539 849427 170527 206046 982226 450103 422307 785790 908945 356231 807886 977097 45932 953670 421492 873747 115983 51635 30257 228258 518910 598328 907107 707570 695831 635715 60774 827717 454064 12520 492333 545837 210491 271267 544651 390286 442167 103972 290745 404250 763661 545442 571479 267564 443149 343797 688062 60702 901754 323706 912806 810570 49783 14874 59307 573895 123177 847415 34409 196884 731864 583330 412778 212896 110296 23933 18283 647186 679103 405668 607715 307978 413628 440550 19851 499540 839399 682262 127157 594130 772340 484247 787355 546420 625841 198295 505364 322024 169086 128751 574485 613050 577821 491089 851121 364025 890894 757502 341002 173072 179676 591376 417922 335234 636325 821478 64575 9753 468878 959987 471918 486601 487898 737716 174504 188173 690970 601036 506811 902593 261951 416973 613005 844020 914711 236722 895971 389098 447732 209955 839578 726377 515773 613355 188972 981596 992491 743848 106812 525412 302695 270603 95599 828652 499301 407028 24051 477377 89672 678160 900152 757551 404063 832646 567520 821035 463965 324702 703073 121408 490805 737133 688058 293633 897938 515065 274913 405670 56298 265423 835747 618371 599858 300332 942965 470834 662703 367182 644174 282247 843537 996828 935334 577375 937858 817334 125934 60652 536018 732734 48556 798592 557065 443044 350918 697460 781874 642236 606116 514556 937259 889986 654006 693409 588683 638397 175882 997416 113204 646348 129406 548259 485003 719696 387676 102832 17210 668533 81765 742054 143425 898106 872686 542504 217813 788219 207412 248301 387346 488823 110553 916997 954754 129543 297889 362838 564197 829494 93308 742789 654354 627344 155742 482149 279165 498509 109599 419170 624510 395776 136977 287975 611253 891147 246433 165349 573093 264367 458342 198 67368 156418 663442 462274 898770 183596 649364 373711 183032 519191 188083 932936 785591 363563 959139 701066 217641 602129 35875 935640 521190 711350 934137 949177 737457 495638 161565 229034 328445 189816 69737 741687 537724 404013 942732 30700 266318 934068 599197 357478 292306 99929 714935 272610 968886 333479 399394 279052 263316 561995 595420 48743 425382 278570 625310 671563 419756 896864 833132 334510 410065 752943 228785 413947 388174 553795 373402 547280 923736 77104 122742 189373 817791 37319 3979 106856 80399 479182 438502 148941 307113 757024 840487 674581 296757 146133 618088 137492 93900 79119 572391 100924 315916 96490 460207 518079 721951 110067 925008 316648 879551 820528 750059 35694 35048 698954 649494 153123 778474 140419 159909 214594 568452 107300 413105 312276 94264 721836 241388 624961 357196 928649 894462 427982 63990 313992 102405 779041 962090 954311 984123 540002 470877 859927 102067 671322 363990 301205 690821 96476 58369 986244 255026 457988 912396 782823 669036 854926 377373 71150 513723 451614 694444 602360 461969 98766 807854 402832 325471 371108 828837 746291 572612 983005 528317 691308 560000 277916 989481 251959 551224 142620 569108 378309 847420 837375 880051 418674 414226 597135 385596 