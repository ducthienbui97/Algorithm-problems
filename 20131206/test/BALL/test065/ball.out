15072506680290
